tools/build-neatvnc.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Update and install dependencies
+sudo apt update
+sudo apt install -y meson ninja-build libpixman-1-dev zlib1g-dev libdrm-dev pkgconf
```

Comment:
You should install `pkg-config` instead.
See https://packages.ubuntu.com/search?keywords=pkg-config